sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -91,6 +111,21 @@ export class AppConfigurationClient {
     }
   }
 
+  /**
+   * Add a setting into the Azure App Configuration service, failing if it 
+   * already exists.
+   * 
+   * Example usage:
+   * ```ts
+   * const key = "MyKey";
+   * const label = "MyLabel";
+   * const value = "MyValue";
+   * const result = await client.addConfigurationSetting(key, { label, value });
```

Comment:
I would collapse the 4 line example into a single line by inlining the arguments as the emphasis is on `addConfigurationSetting`.
Also, in editors this would occupy less space and would require no scrolling

Reply:
Yes, agree. The keys used to be things like 'foo' and such and they wren't descriptive enough. Not a problem now.